Core/Core/API/APIActivity.swift

```diff
@@ -37,7 +37,7 @@ public struct APIActivity: Codable {
     let created_at: Date
     let updated_at: Date
     let type: ActivityType
-    let context_type: String
+    let context_type: String?
```

Comment:
does `context_type` not always come back?

Reply:
yeah, i caught one that did not.